src/applications/caregivers/actions/fetchFacilities.js

```diff
@@ -67,8 +67,15 @@ export const fetchFacilities = async ({
 
   return fetchRequest
     .then(response => {
-      return response.map(facility => {
-        const { physical } = facility?.address;
+      if (!response?.data?.length) {
```

Comment:
This should not really happen now, but I added logic to handle if the data object is not returned (this can happen on the backend if the pagination values are passed in out of range). If you try to send the searchbox garbage (ie search for 'my city` or something it will generally return this same response but from the mapbox api call).